packages/nocodb/src/middlewares/extract-ids/extract-ids.middleware.ts

```diff
@@ -180,8 +180,8 @@ export class ExtractIdsMiddleware implements NestMiddleware, CanActivate {
 }
 
 function getUserRoleForScope(user: any, scope: string) {
-  if (scope === 'project' || scope === 'workspace') {
-    return user?.project_roles || user?.workspace_roles;
+  if (scope === 'project') {
```

Comment:
1. Why we don't need to handle workspace? 
2. What if the scope is neither `project` nor `org`? What is the default return here?

Reply:
1. We don't have workspace concept here
2. There is only `org` and `project` scope at the moment

Reply:
- ws is not present for os
- scope is default to 'project', for others we specify via @Acl(perm, { scope: 'org' })